index.html

```diff
@@ -1,12 +1,216 @@
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="index.html">Home</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="about.html">About</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">Program</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">Join</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">Sponsor</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">News</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">My page</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">Logout</a>
+          </li>
+        </ul>
+      </div>
+      <!-- top bar -->
+      <div class="top-bar flex-row">
+        <a href="#"><img src="./assets/img/facebook.png" alt="facebook icon" /></a>
+        <a href="#"><img src="./assets/img/twitter.png" alt="twitter icon"/></a>
+        <a href="#" class="text-12-LH-14">English</a>
+        <a href="#" class="text-12-LH-14">My Page</a>
+        <a href="#" class="text-12-LH-14">Logout</a>
+      </div>
+      <!-- Menu items -->
+      <nav class="">
+        <!-- Hamburger-->
+        <img
+          id="hamburger"
+          class="hamburger"
+          src="./assets/img/mobile-menu-1.png"
+          alt="hamburger icon"
+        />
+        <!-- LOGO -->
+        <a href="index.html">
+          <img class="logo" src="./assets/img/logo.png" alt="logo" />
+        </a>
+        <ul>
+          <a href="about.html" class="text-14-bold">About</a>
+          <a href="#" class="text-14-bold">Program</a>
+          <a href="#" class="text-14-bold">Join</a>
+          <a href="#" class="text-14-bold">Sponsor</a>
+          <a href="#" class="text-14-bold">News</a>
+          <!-- Action button -->
+          <a href="#" class="action-btn-1 text-14-bold">WD Campaign</a>
+        </ul>
+      </nav>
+    </header>
+    <!-- Hero section -->
+    <section class="hero-container flex-column">
+      <div class="hero flex-column">
+        <div class="hero-title">
+        </div>
+        <p class="sub-theme flex-row text-12 big-text-20 custom-gray-2">
+          A joyful celebration believing in the value of openness and sharing,
+          creating a positive change with people from over 80 countries is
+          taking place in October, in Korea.
+        </p>
+        <div class="date-venue flex-column">
+          <h1 class="custom-gray-3 text-20-bold big-text-42">2023.10.19(THU) ~ 20(FRI)</h1>
+          <p class="custom-gray-4 text-13 big-text-27">
+            @ National Museum of Karea, Art Center Nabi and more
+          </p>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] You did great implementing the changes suggested by the previous reviewer in this section. :smile: :+1: But I noticed some responsiveness issues you need to fix. On mobile devices, the main section content tends to overflow into the next section, PLease you need to fix this as doesn't look great in the UI and also doesn't give the best user experience. :+1: 
- YOUR DESIGN:
![Selection_017](https://user-images.githubusercontent.com/107632780/234996968-dbf5425f-9526-4461-977c-86e067b427f0.png)

- FIGMA DESIGN:

![Selection_016](https://user-images.githubusercontent.com/107632780/234997047-dba21947-93ac-4a3c-9622-8639603be9aa.png)

Reply:
@V-Blaze this renders fine on my end and all screens. To help me find the bug, I need you to tell me what platform this is coming from and screen-size please. Thanks

Reply:
> @V-Blaze this renders fine on my end and all screens. To help me find the bug, I need you to tell me what platform this is coming from and screen-size please. Thanks

Hello... @DebyGrey , The screenshot was taken from the deployed version of your project , the link from your readme file.. check any mobile screen size using the chrome dev tool. 😄

Reply:
@V-Blaze okay thanks for the direction. I am looking at it now but I think pages might have bugs or cache problems though because it keeps rendering wrong and this won't be the first time I'm experiencing such. It works fine on my local server. I fear that I might go ahead to scatter my codes.

Reply:
@V-Blaze I found the issue. The problem is from pages. I don't think it reads 2-3 level folder structure because when I replaced ../assets/img/speaker-01.png with ./assets/img/speaker-01.png it worked. Meanwhile, on my local server, everything works fine. Just thought I'd share.